chain/client/src/client_actor.rs

```diff
@@ -1372,22 +1363,29 @@ impl ClientActor {
         })
     }
 
-    fn receive_headers(&mut self, headers: Vec<BlockHeader>, peer_id: PeerId) -> bool {
+    fn receive_headers(&mut self, mut headers: Vec<BlockHeader>, peer_id: PeerId) -> bool {
         let _span =
             debug_span!(target: "client", "receive_headers", num_headers = headers.len(), ?peer_id)
                 .entered();
         if headers.is_empty() {
+            info!(target: "client", "Received no block headers");
             return true;
         }
-        info!(target: "client", "Received block headers from height {} to {}", headers.first().unwrap().height(), headers.last().unwrap().height());
+        // Sort headers by heights.
+        headers.sort_by_key(|left| left.height());
```

Comment:
nit: do we want to sort them twice?
The only reason to sort them here is to print `info` message.
We have almost the same `debug` message in `sync_block_headers`.
I'd prefer not to add `mut`, drop this `info` message, change logging level from `debug` to `info` in `sync_block_headers`